README.md

```diff
@@ -40,7 +48,7 @@ In an AJAX POST request, what does the argument of the `.done()` callback - in t
   });
 ```
 ### Question 6
-Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`. 
+Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`.
```

Comment:
Looks like you didn't get to Q6. You need to find the text box with ID of 'color', which is where the user typed stuff, and then use the `.val()` method to get what they typed. So `var user_input = $('#color').val()`.